pkgs/development/python-modules/fastcrc/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchPypi,
+  rustPlatform,
+  nix-update-script,
+}:
+let
+  pname = "fastcrc";
+  version = "0.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Tip1NZ2XpZRpu8vXdRPoAdPzZmqRaWKVePiAEsDy1/4=";
+  };
+in
+python3.pkgs.buildPythonPackage {
```

```suggestion
buildPythonPackage {
```
Please use `buildPythonPackage` directly, since `python3.pkgs.buildPythonPackage` may cause a python version mismatch.
I mean, `nix-build -A python311Packages.fastcrc` causes the following error.
```
error: fastcrc should use `buildPythonPackage` or `toPythonModule` if it is to be part of the Python packages set.
```